mysql-test/suite/rocksdb/r/rocksdb.result

```diff
@@ -879,7 +879,7 @@ rocksdb_compaction_sequential_deletes	0
 rocksdb_compaction_sequential_deletes_count_sd	OFF
 rocksdb_compaction_sequential_deletes_file_size	0
 rocksdb_compaction_sequential_deletes_window	0
-rocksdb_concurrent_prepare	ON
+rocksdb_two_write_queues	ON
```

Comment:
The list looks sorted. Does the testcase pass?

Reply:
Thanks. I just noticed the test failure. I rushed into submitting the PR. I will submit a fix soon.